src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/utils/VisitBalancesUtil.kt

```diff
@@ -5,25 +5,36 @@ import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.pri
 import java.time.LocalDate
 
 @Component
-class VisitBalancesUtil {
+class VisitBalancesUtil(private val dateUtil: DateUtil) {
   fun calculateAvailableVos(visitBalance: VisitBalancesDto?): Int {
     return (visitBalance?.remainingVo ?: 0) + (visitBalance?.remainingPvo ?: 0)
   }
 
-  fun calculateVoRenewalDate(visitBalance: VisitBalancesDto?, dateFrom: LocalDate = LocalDate.now()): LocalDate {
-    val latestIepAdjustDate = calculateIepAdjustDate(visitBalance, dateFrom)
-    val latestPrivilegedIepAdjustDate = calculatePrivilegedIepAdjustDate(visitBalance, dateFrom)
+  fun calculateVoRenewalDate(visitBalance: VisitBalancesDto?): LocalDate {
+    val currentDate = dateUtil.getCurrentDate()
+    val latestIepAdjustDate = calculateIepAdjustDate(visitBalance, currentDate)
```

Comment:
is this latestIepAdjustDate a fancy way of saying VO

Reply:
it is what comes from prison API but I could possibly rename that on our end to a name that suits us.

Reply:
so could we call it latestVoAdjustedDate or may be latestAdjustedDate

Reply:
updated variable names